botorch/acquisition/input_constructors.py

```diff
@@ -1780,7 +1780,7 @@ def optimize_objective(
         bounds=free_feature_bounds,
         q=q,
         num_restarts=optimizer_options.get("num_restarts", 60),
-        raw_samples=optimizer_options.get("raw_samples", 1024),
+        raw_samples=optimizer_options.get("raw_samples", 1024),  # NOTE potential behaviour change
```

Comment:
Not sure I understand the NOTE here. Is this in the right place? Did you mean to change the code to return a `None` default here?

Reply:
The note here is just a reminder to ask about this edge case in the review :)

Elsewhere in the code the default is indeed None whereas here it defaults to a value. If defaulted to None then there would be no behaviour change for someone giving initial batch conditions but since it defaults to a value if the user gave less than 1024 initial conditions then they would see a behaviour change.